render/mapping.go

```diff
@@ -276,7 +307,10 @@ func MapContainer2ContainerImage(n RenderableNode) (RenderableNode, bool) {
 		return n, false
 	}
 
-	return newDerivedNode(id, n), true
+	// Add container-<id> key to NMD, which will later be counted to produce the minor label
+	result := newDerivedNode(id, n)
+	result.NodeMetadata.Metadata[containersPrefix+n.ID] = ""
+	return result, true
```

Comment:
> This is 'the mapreduce way'. Emit unique keys from the mapper and then count them in the reducer.

It's the wordcount way. But we're not reducing text files, we're semantically merging dicts. A key with no value feels like a yellow flag.

> Are you suggesting the container_count be a field on NodeMetadata? Or a key in the NodeMetadata.Metadata map?

Either one.

> A more generic solution would be to add a map[string]int field to the RenderableNode struct called counters, which was not json serialised, and define the merge semantics on that. But that seems like massive overkill for these two uses.

I agree, that would be premature generalization.